examples/verifying_mfd.py

```diff
@@ -1,16 +1,183 @@
+          "the appartments differ by 0.000229 in Longitude and by 0.00004 "
+          "in Latitude, with the 2 points being merely "
+          "around 0.01979 km (or 0.012298 miles) apart, "
+          "which is considered to be the same place with "
+          "parameter δ = 0.001, but violates MFD with parameter δ = 0.0001.")
+    print("-"*80)
+
+def mfd_basic(table="examples/datasets/theatres_mfd.csv"):
+    METRIC = "euclidean"
+    LHS_INDICES = [0]
+    RHS_INDICES = [2]
+    PARAMETERS = [5, 3]
+
+    print(f"{BLUE_CODE}Metric Functional Dependency{DEFAULT_COLOR_CODE} "
+          "(MFD) is a type of functional "
+          "dependency designed to combat small deviations that would "
+          "otherwise invalidate regular functional dependencies. Those "
+          "deviations are measured using arbitrary metric defined by user, "
+          " making this definition applicable in a variety of situations.\n")
+    print("Semi-formal definition for those interested:")
+    print("Given a parameter δ and metric Δ, an MFD is assumed to hold if "
+          "maximum of Δ(x, y) <= δ, where x and y are different right-hand "
+          "sides sharing the same left-hand side.\n")
+    print("Let's start by investigating the theatres_mfd.csv and "
+          "trying to verify the metric functional dependency in it.\n")
+
+    data = pd.read_csv(table, header=[0])
+    print(data)
+
+    for value in PARAMETERS:
+        algo = desbordante.mfd_verification.algorithms.Default()
+        algo.load_data(table=(table, ",", True))
+        algo.execute(lhs_indices=LHS_INDICES, metric=METRIC,
+                                parameter=value, rhs_indices=RHS_INDICES)
+        print("Checking whether [Movie] -> [Duration] AFD holds with "
+              f"parameter δ = {value}")
+        print_results(algo, data, value)
+
+    print("We learnt that in this case, the specified MFD does not hold, and "
+          "there are two clusters of rows that contain values that prevent "
+          f"our FD from holding. A {BLUE_CODE}cluster{DEFAULT_COLOR_CODE} "
+          "(with respect to a fixed FD) is a collection of rows that share "
+          "the same left-hand side, but differ in the right-hand side.")
+    print("Let's take a closer look at the indicated clusters.\n")
+    print("Don Quixote has been found with 3 different durations: 135, 139 "
+          "and 140. From this, we can conclude that "
+          "(assuming euclidian distance), the maximum distance between "
+          "values is 140 - 135 = 5, which exceeds provided "
+          "parameter of δ = 3. When we tried to verify the same MFD on this "
+          "dataset with δ = 5, this cluster adhered to the MFD as 5 <= δ, "
+          "and the whole MFD held.")
+    print("Romeo and Juliet had the exact same issue- the biggest "
+          "difference between durations of different versions appeared to "
+          "have been 165 - 160 = 5, which held with bigger parameter value.")
```

Comment:
```suggestion
          "have been 165 - 160 = 5, which held with a ? parameter value.")
```
I'm not sure if "bigger" works here but can't find an alternative.

Reply:
"held with greater parameter value"